failing_tests/test_code.py

```diff
@@ -18,18 +18,20 @@ def test_multipliers(self):
         
         multipliers = []
         for val in range(6):
-            multiplier = lambda x: x * val
-            multipliers.append(multiplier)
+            def func(val):
+                return lambda x: x * val            

```

Comment:
Since we are using python3.6 so print is a function write 'braces' Since we are using python3.6 so print is a function write 'braces'